geomstats/geometry/euclidean.py

```diff
@@ -63,6 +63,40 @@ def exp(self, tangent_vec, base_point):
         """
         return tangent_vec + base_point
 
+    @property
+    def is_connected(self):
+        """Indicates whether the geometric object is connected.
+
+        Returns
+        -------
+        bool
+            Always True for Euclidean spaces, as they are path-connected.
+            Any two points in a Euclidean space can be joined by a straight
+            line segment.
+
+        References
+        ----------
+        .. [1] James Munkres, "Topology," 2nd edition, page 155.
```

Comment:
can you please complete the reference (e.g. missing year). plus, `[1] -> [M2014]` for style consistency (it may be still inconsistent in parts of the library, but we're trying to stick to this style).

same for other references.